### PR TITLE
MOTECH-2693 Added one more module in whitch we have to stop some bundles

### PR DESCRIPTION
**Pull request by PJSosnowski**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by martokarski** [comment]
test this please

File: platform/mds/mds/src/main/java/org/motechproject/mds/util/Constants.java [hunk 1]
```diff
@@ -482,10 +482,11 @@ private Util() {
...
 
-        public static final String WEB_SECURITY_MODULE = "MOTECH Web Security";
-        public static final String SERVER_CONFIG_MODULE = "MOTECH Platform Server Config";
-        public static final String SCHEDULER_MODULE = "MOTECH Scheduler";
-
+        public static final String WEB_SECURITY_MODULE = "org.motechproject.motech-platform-web-security";
+        public static final String SERVER_CONFIG_MODULE = "org.motechproject.motech-platform-server-config";
+        public static final String SCHEDULER_MODULE = "org.motechproject.motech-platform-scheduler";
+        public static final String TASKS_MODULE = "org.motechproject.motech-tasks";
+        public static final String EMAIL_MODULE = "org.motechproject.motech-email";
```
Comment (kmadej) [inline platform/mds/mds/src/main/java/org/motechproject/mds/util/Constants.java]: do you use this EMAIL_MODULE constant ?

File: platform/mds/mds/src/main/java/org/motechproject/mds/service/impl/JarGeneratorServiceImpl.java [hunk 2]
```diff
@@ -636,6 +641,9 @@ private void waitForPackagesRefreshed() {
                 }
                 ++count;
             }
+            if(count >= MAX_WAIT_COUNT) {
```
Comment (kmadej) [inline platform/mds/mds/src/main/java/org/motechproject/mds/service/impl/JarGeneratorServiceImpl.java]: space after if

File: platform/mds/mds/src/main/java/org/motechproject/mds/service/impl/JarGeneratorServiceImpl.java [hunk 3]
```diff
@@ -636,6 +641,9 @@ private void waitForPackagesRefreshed() {
...
+                LOGGER.error("Package has not been refreshed, reached time limit");
```
Comment (kmadej) [inline platform/mds/mds/src/main/java/org/motechproject/mds/service/impl/JarGeneratorServiceImpl.java]: Packages have not been ...

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
test this please

**Pull request merged**